woocommerce/woocommerce-android
# Store creation help button

Fixes #8046.

This PR refactors the usage of `Toolbar` Composable and the navigation to the Help screen and adds the help button to a couple of store creation screens.

I noticed the same code for the `Toolbar` keeps getting copied over almost in every Compose screen, so I extracted it to its own reusable component and removed the duplicate code from all the screens I found. The same applied to the `NavigateToHelpScreen` event and the associated navigation code in the fragments.

<img src="https://user-images.githubusercontent.com/1522856/208245308-f99724d7-3747-4a4d-ab22-c962de46de92.png" width="300" />

**To test:**
1. Start the store creation flow from the site picker
2. Notice there is now a help button on the store name screen
3. Verify it navigates to the Help screen
4. Go to the domain picker screen and verify the help button works
5. Go to the store summary and verify the same thing
